docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -7,17 +7,17 @@ tags:
 
 # Why aren't findings populating in the GitHub Advanced Security Dashboard after running Semgrep in CI?
 
-When scanning with Semgrep in CI, findings automatically populate in Semgrep Cloud Platform. To show findings in the GitHub Advanced Security Dashboard, you must first run an alternate job that uploads findings in the form of a `SARIF` file to the dashboard. See  [Sample GitHub Actions configuration file](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file) for an example.
+When scanning with Semgrep in CI, findings automatically populate in Semgrep Cloud Platform. To show findings in the GitHub Advanced Security Dashboard, run an alternate job that uploads findings to the dashboard in the form of a `SARIF` file. See  [Sample GitHub Actions configuration file](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file) for an example.
 
 If you run the alternate job and it fails with a "resource not accessible by integration" error, there are two possible causes.
 
 ## Your repository is private
```

Comment:
TIL! I had no idea our GHAS export had a "public repos only" limitation! Dumb question, but third-party code scanning findings - I thought of SSC, but isn't this for SAST?

Reply:
Yes, you're right, that's a confusing term. I think in this case it's meant to indicate that Semgrep is a third-party where GHAS is concerned. @pjzcodes IIRC you wrote this, can you confirm?